package.json

```diff
@@ -1,29 +1,64 @@
 {
   "name": "@quixo3/prisma-session-store",
   "description": "Prisma express-session store",
-  "keywords": ["Prisma", "Express", "Session", "Memory", "Store"],
-  "version": "0.2.0",
+  "keywords": [
+    "Prisma",
+    "Express",
+    "Session",
+    "Memory",
+    "Store",
+    "TypeScript"
+  ],
+  "version": "0.3.0",
   "license": "MIT",
   "author": "Krispin Leydon",
+  "contributors": [
+    "wSedlacek"
+  ],
   "repository": {
     "type": "git",
     "url": "https://github.com/kleydon/prisma-session-store.git"
   },
-  "main": "index.js",
+  "bugs": {
+    "url": "https://github.com/kleydon/prisma-session-store/issues"
+  },
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist",
+    "README.md",
+    "LICENSE"
+  ],
   "scripts": {
-    "test": "mocha --check-leaks --bail --no-exit test/"
+    "test": "jest",
```

Comment:
The `@sanpuru/prisma-session-store` package is just the compiles files from `tsc` (Note the `files` key in `package.json` that controls what is published)


To test this package run the following commands.

```sh
git clone https://github.com/wSedlacek/prisma-session-store.git -b typescript
yarn install
yarn test
```

Reply:
@wSedlacek Thanks for the additional testing notes. All tests passed for me (as you knew they would!)

Testing made me wonder whether it should be possible to test at the dist/ level, rather than (or in addition to) the src/ level - since it is the dist/ code that will actually be used in production... Do you know if this is possible? Maybe its something to consider for the future...

Reply:
It is possible to test at the dist level, however it is not typically done.
The `dist` is a direct result of the `src` so if it runs in `src` there is no chance it would not run in `dist`
When `ts-jest` runs it has to compile the `src` down into JavaScript (which it does in memory) which is effectively the same as when `tsc` transpiles to the `dist` folder